### PR TITLE
Prompt utils

### PR DESCRIPTION
**Pull request by harry-humanloop**
- Add `humanloop.prompts.populate_template`

image

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "humanloop"
-version = "0.8.6b3"
```
Comment (peadaroh) [inline pyproject.toml:3]: @harry-humanloop was this intentional?

File: src/humanloop/eval_utils.py [hunk 2]
```diff
@@ -208,29 +214,29 @@ def _run_eval(
...
-        
+
     elif type_ == "prompt":
         try:
             _ = Prompt.parse_obj(version)
         except ValidationError as error_:
             logger.error(msg=f"Invalid Prompt `version` in your `file` request. \n\nValidation error: \n)")
             raise error_
         hl_file = client.prompts.upsert(**file_dict)
-
```
Comment (peadaroh) [inline src/humanloop/eval_utils.py:219]: Are these ruff changes?

File: pyproject.toml (hunk 1 above)
Comment (harry-humanloop) [reply pyproject.toml:3]: not really; this was from the first commit in this PR, by fern-bot, which i suspect comes from `fern generate`

(https://github.com/humanloop/humanloop-python/pull/21/commits/77b76f7dcd91cec808fdb6b7c4ae59cc3eac7799)

File: src/humanloop/eval_utils.py (hunk 2 above)
Comment (harry-humanloop) [reply src/humanloop/eval_utils.py:219]: all are ruff, apart from https://github.com/humanloop/humanloop-python/pull/21/commits/deefc8b45d89b9b0de7ef41a9659881d460770fc

**Pull request merged**